### PR TITLE
Content/interview christoffer stromberg

### PR DESCRIPTION
**Pull request by strobba**
interview added

### CODE REVIEW

**Review by docNord — CHANGES_REQUESTED** [review]
Great answers!
- [x] Send picture file to @mgcth for storage
- [ ] fix lint (or I can do it later if it's a hazzle)
and we can get it up!

**Comment by mgcth** [comment]
Picture is up, please change this filename to `intervju-christoffer-stromberg.md`, image filename is `christoffer_stromberg.jpeg`.

**Review by docNord — APPROVED** [review]

**Comment by mgcth** [comment]
Change the filename...

File: content/posts/interview-christoffer-stromberg.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+title: Intervju med medlem: Christoffer Strömberg
+date: 2023-03-26
+author: Christoffer Strömberg
+category: Om
+tags: intervju, klimat, ifk, medlemmar
+status: unpublished
```
Comment (mgcth) [inline content/posts/interview-christoffer-stromberg.md:6]:
```suggestion
status: published
```

**Review by mgcth — CHANGES_REQUESTED** [review]
Change filename to `intervju-christoffer-stromberg.md`.

File: content/posts/interview-christoffer-stromberg.md (hunk 1 above)
Comment (strobba) [reply content/posts/interview-christoffer-stromberg.md]: done and done

**Review by mgcth — APPROVED** [review]
🚀

**Pull request merged**